.env.example

```
@@ -49,7 +49,7 @@ ACESTEP_LM_BACKEND=vllm
 #
 # Default: auto (based on GPU VRAM detection)
 ACESTEP_INIT_LLM=auto
-
+ACESTEP_FORCE_FLAC_EXPORT=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ACESTEP_FORCE_FLAC_EXPORT key should go before the ACESTEP_INIT_LLM key

(UnorderedKey)